filament/src/Renderer.cpp

```diff
@@ -572,6 +639,26 @@ void FRenderer::readPixels(uint32_t xoffset, uint32_t yoffset, uint32_t width, u
     driver.readPixels(mRenderTarget, xoffset, yoffset, width, height, std::move(buffer));
 }
 
+RenderPass::CommandTypeFlags FRenderer::getCommandType(View::DepthPrepass prepass) const noexcept {
+    RenderPass::CommandTypeFlags commandType;
+    switch (prepass) {
+        case View::DepthPrepass::DEFAULT:
+            // TODO: better default strategy (can even change on a per-frame basis)
+            commandType = RenderPass::DEPTH_AND_COLOR;
+#if defined(ANDROID) || defined(__EMSCRIPTEN__)
```

Comment:
Maybe also check if we're on iOS here too? (`defined(IOS)`)

Reply:
Good point -- I'll do this in another PR. I think the test should actually be "mobile" vs. "desktop". We don't have such a compile flag yet.